Various configuration including:

Cached result of ./configure --disable-seabios --enable-lomount

diff -r 9ac9766410e4 .config
--- /dev/null
+++ b/.config
@@ -0,0 +1,8 @@
+# Do no build Qemu as part of Xen
+CONFIG_IOEMU := n
+
+# Perform a debug build while integrating Xen-4.2 into trunk
+debug := y
+# debug_symbols := y
+
+LIBXL_BLKTAP := n
diff -r 9ac9766410e4 config/Tools.mk
--- /dev/null
+++ b/config/Tools.mk
@@ -0,0 +1,61 @@
+# Prefix and install folder
+prefix              := /usr
+PREFIX              := $(prefix)
+exec_prefix         := ${prefix}
+libdir              := ${exec_prefix}/lib
+LIBDIR              := $(libdir)
+
+# A debug build of tools?
+debug               := y
+
+# Tools path
+BISON               := /usr/bin/bison
+FLEX                := /usr/bin/flex
+PYTHON              := python
+PYTHON_PATH         := /usr/bin/python
+PERL                := /usr/bin/perl
+CURL_CONFIG         := 
+XML2_CONFIG         := 
+BASH                := /bin/sh
+XGETTTEXT           := /usr/bin/xgettext
+AS86                := /usr/bin/as86
+LD86                := /usr/bin/ld86
+BCC                 := /usr/bin/bcc
+IASL                := /usr/bin/iasl
+
+# Extra folder for libs/includes
+PREPEND_INCLUDES    := 
+PREPEND_LIB         := 
+APPEND_INCLUDES     := 
+APPEND_LIB          := 
+
+PTHREAD_CFLAGS      := -pthread
+PTHREAD_LDFLAGS     := -pthread
+PTHREAD_LIBS        := 
+
+PTYFUNCS_LIBS       := -lutil
+
+# Download GIT repositories via HTTP or GIT's own protocol?
+# GIT's protocol is faster and more robust, when it works at all (firewalls
+# may block it). We make it the default, but if your GIT repository downloads
+# fail or hang, please specify GIT_HTTP=y in your environment.
+GIT_HTTP            := n
+
+# Optional components
+XENSTAT_XENTOP      := y
+VTPM_TOOLS          := n
+LIBXENAPI_BINDINGS  := n
+OCAML_TOOLS         := y
+CONFIG_MINITERM     := n
+CONFIG_LOMOUNT      := y
+CONFIG_OVMF         := n
+CONFIG_ROMBIOS      := y
+CONFIG_SEABIOS      := n
+
+#System options
+CONFIG_SYSTEM_LIBAIO:= y
+ZLIB                :=  -DHAVE_BZLIB -lbz2
+CONFIG_LIBICONV     := n
+CONFIG_GCRYPT       := y
+EXTFS_LIBS          := -lext4fs
+CURSES_LIBS         := -lncurses
diff -r 9ac9766410e4 tools/config.h
--- /dev/null
+++ b/tools/config.h
@@ -0,0 +1,71 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#define HAVE_INTTYPES_H 1
+
+/* Define to 1 if you have the `crypto' library (-lcrypto). */
+#define HAVE_LIBCRYPTO 1
+
+/* Define to 1 if you have the `yajl' library (-lyajl). */
+#define HAVE_LIBYAJL 1
+
+/* Define to 1 if you have the `z' library (-lz). */
+#define HAVE_LIBZ 1
+
+/* Define to 1 if you have the <memory.h> header file. */
+#define HAVE_MEMORY_H 1
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#define HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#define HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#define HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#define HAVE_STRING_H 1
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#define HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#define HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#define HAVE_UNISTD_H 1
+
+/* Define to 1 if you have the <yajl/yajl_version.h> header file. */
+#define HAVE_YAJL_YAJL_VERSION_H 1
+
+/* Define curses header to use */
+#define INCLUDE_CURSES_H <ncurses.h>
+
+/* Define extfs header to use */
+#define INCLUDE_EXTFS_H <ext4fs/ext2fs.h>
+
+/* libutil header file name */
+/* #undef INCLUDE_LIBUTIL_H */
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT "xen-devel@lists.xen.org"
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME "Xen Hypervisor"
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING "Xen Hypervisor 4.2"
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME "xen-hypervisor"
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL ""
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "4.2"
+
+/* Define to 1 if you have the ANSI C header files. */
+#define STDC_HEADERS 1
diff -r 9ac9766410e4 xen/xen-version
--- /dev/null
+++ b/xen/xen-version
@@ -0,0 +1,3 @@
+# Replace - with . to make version suitable for rpm.
+XEN_EXTRAVERSION := $(subst -,.,$(XEN_EXTRAVERSION))
+XEN_FULLVERSION  := $(subst -,.,$(XEN_FULLVERSION))
